sympy/matrices/matrices.py

```diff
@@ -1553,53 +1541,354 @@ def eigenspace(eigenval):
             # non-trivial.  If we didn't find an eigenvector, try once
             # more a little harder
             if len(ret) == 0 and simplify:
-                ret = m.nullspace(iszerofunc=iszerofunc, simplify=True, dotprodsimp=dotprodsimp)
+                ret = m.nullspace(iszerofunc=iszerofunc, simplify=True,
+                        dotprodsimp=dotprodsimp)
             if len(ret) == 0:
                 raise NotImplementedError(
                         "Can't evaluate eigenvector for eigenvalue %s" % eigenval)
             return ret
 
+        simplify   = flags.get('simplify', True)
+        simpfunc   = simplify if isinstance(simplify, FunctionType) else \
+                _simplify if simplify else lambda x: x
+        primitive  = flags.get('simplify', False)
+        chop       = flags.pop('chop', False)
+        mat        = self
+        has_floats = self.has(Float)
+
+        flags.pop('multiple', None)  # remove this if it's there
+
+        # roots doesn't like Floats, so replace them with Rationals
+        if has_floats:
+            mat = mat.applyfunc(lambda x: nsimplify(x, rational=True))
+
         eigenvals = mat.eigenvals(rational=False,
                                   error_when_incomplete=error_when_incomplete,
                                   dotprodsimp=dotprodsimp,
                                   **flags)
+
+        # if (mat.rows > 3 and all(m == 1 for m in eigenvals.values()) and
+        #         all(e.is_real is True for e in mat) and
+        #         mat.is_symmetric()):
+        #     # real symmetric matrices larger than order 3 with multiplicity 1
+        #     # eigenvalues can be evaluated faster via ``_eigenvects_by_minors``
+        #     return self._eigenvects_by_minors(iszerofunc=iszerofunc, simplify=simplify,
+        #             dotprodsimp=dotprodsimp, chop=chop, has_floats=has_floats)
```

Comment:
Please don't add any commented-out codes or tests to the repo. This is not your own sketchbook.
But try to open up a new issue thread to if you want to pick your stuff up

Reply:
Its there just so you can look at it, this is not a finished PR it will be removed don't worry... I just want confirmation from you guys that we drop `_eigenvects_by_minors` and keep `eigenvects_mod_square_elementwise`.